Release.toml

```diff
@@ -234,7 +234,8 @@ version = "1.16.0"
     "migrate_v1.15.0_log4j-hotpatch-enabled-metadata.lz4",
     "migrate_v1.15.0_deprecate-log4j-hotpatch-enabled.lz4",
 ]
-"(1.15.0, 1.16.0)" = [
+"(1.15.0, 1.15.1)" = []
```

Comment:
Are you planning to do another PR to update the version in this file, at the very top of this file?

Reply:
Yes, that needs to be done only on the 1.15.x branch.